ui/public/i18n/language.json

```diff
@@ -429,7 +429,10 @@
         "udpPorts": "UDP ports",
         "access": "Access",
         "service_edited_successfully": "Service edited successfully",
-        "service_edited_error": "Service not edited"
+        "service_edited_error": "Service not edited",
+        "Order": "Rule position",
+        "top": "create at the top",
```

Comment:
do we use the capital letter? `Create at the top`

Reply:
I suggest to reuse the same labels from NethGUI: https://github.com/NethServer/nethserver-firewall-base/blob/master/root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_FirewallRules.php#L43-L44

Also, I'd change the label key name from `Order` to `order`: all other keys are lowercase :)

Reply:
done